packages/firebase_remote_config/firebase_remote_config_platform_interface/lib/src/remote_config_settings.dart

```diff
@@ -0,0 +1,8 @@
+class RemoteConfigSettings {
+
+  /// docs
+  RemoteConfigSettings(this.fetchTimeout, this.minimumFetchInterval);
```

Comment:
With this API the code to configure looks like this:

```dart
  remoteConfig.setConfigSettings(RemoteConfigSettings(
      Duration(seconds: 10), Duration(hours: 0)
  ));
```
  
So you would only see two durations and always would have to guess the parameters or look it up (every time).
  
I would prefer an API like this:
 ```dart
   remoteConfig.setConfigSettings(RemoteConfigSettings(
      fetchTimeout: Duration(seconds: 10),
      minimumFetchInterval: Duration(hours: 0),
  ));
```

This just makes it so much more visible to the API user what the current values mean.

---

For reference, this is the API on the other platforms:

Web:
```js
remoteConfig.settings.fetchTimeoutMillis = 10000;
remoteConfig.settings.minimumFetchIntervalMillis = 0;
```

Android:
```kt
remoteConfig = Firebase.remoteConfig
val configSettings = remoteConfigSettings {
    fetchTimeoutInSeconds = 10
    minimumFetchIntervalInSeconds = 0
}
remoteConfig.setConfigSettingsAsync(configSettings)
```

iOS:
```swift
remoteConfig = RemoteConfig.remoteConfig()
let settings = RemoteConfigSettings()
settings. fetchTimeout = 10
settings.minimumFetchInterval = 0
remoteConfig.configSettings = settings
```
---
Everywhere the APIs are clear, so I would propose:
```suggestion
  RemoteConfigSettings({this.fetchTimeout, this.minimumFetchInterval});
```

Reply:
Done!